torch/csrc/jit/passes/onnx/remove_inplace_ops_for_onnx.cpp

```diff
@@ -307,25 +323,22 @@ Value* MatchIfBlocksOutputForValue(
 
   for (size_t i = 0; i < output_size - 1; i++) {
     if (outer_block->outputs().at(i)->debugNameBase() ==
-        origOutput->debugNameBase()) {
-      outer_block->owningNode()
-          ->outputs()
-          .at(output_size - 1)
-          ->replaceAllUsesWith(outer_block->owningNode()->outputs().at(i));
-      outer_block->owningNode()->eraseOutput(output_size - 1);
+        origOutput->debugNameBase()) { // Check debug names
```

Comment:
would it be possible to create a new data structure to keep track of the relations between values, other than using debug names? just wondering the idea, not strictly something to add for in this PR.